tdrs-backend/tdpservice/users/test/factories.py

```diff
@@ -25,7 +25,7 @@ class Meta:
     login_gov_uuid = factory.Faker("uuid4")
     deactivated = False
     # For testing convenience, though most users won't have both a login_gov_uuid and hhs_id
-
+    nextgen_xid = factory.Faker("uuid4")
```

Comment:
this aughta work in a test, its still a string, but maybe should be updated for clarity